changelog/README.md

```diff
@@ -1,10 +1,21 @@
 # Changelog
 
+## 11th October 2024
+* Deprecated property `BaseRateId` in [Rate](../operations/rates.md#rate). Use `BaseRatePricing` instead.

```

Comment:
You mentioned that you omitted rates, but some changes are included. It would be better to leave it out of this PR, as there was yet another PR merged for Rates, for which I'll have to update the docs.

Reply:
Sorry, Vojta's PR is about Restrictions and not Rates, my bad. What I meant to say is that I omitted Restrictions.